packages/block-library/src/navigation/frontend.js

```diff
@@ -14,6 +14,10 @@ function navigationToggleModal( modal ) {
 	triggerButton.setAttribute( 'aria-expanded', ! isHidden );
 	closeButton.setAttribute( 'aria-expanded', ! isHidden );
 	modal.classList.toggle( 'has-modal-open', ! isHidden );
+
+	// Add a class to indicate the modal is open.
+	const htmlElement = document.querySelector( 'html' );
```

Comment:
FYI, can avoid using `querySelector` here, the same can be achieved through `const htmlElement = document.documentElement;`.

Reply:
Nice, thank you! https://github.com/WordPress/gutenberg/pull/32303